pkg/rdt/rdt.go

```diff
@@ -0,0 +1,90 @@
+// +build linux
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package rdt
+
+import (
+	"fmt"
+
+	"github.com/containerd/containerd/log"
+	"github.com/intel/goresctrl/pkg/rdt"
+	"github.com/pkg/errors"
+
+	"github.com/containerd/containerd/plugin"
```

Comment:
nit: these are grouped weirdly.  Can you do something like this instead?

```suggestion
	"github.com/containerd/containerd/log"
	"github.com/containerd/containerd/plugin"

	"github.com/intel/goresctrl/pkg/rdt"
	"github.com/pkg/errors"
```

This keeps the imports that are part of this module together and the imports coming from other modules separate.

Reply:
Indeed, dunno why it was grouped that way 🧐  Now fixed